tools/restatectl/Cargo.toml

```diff
@@ -63,7 +63,7 @@ strum = { workspace = true }
 tempfile = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
-tonic = { workspace = true, features = ["transport", "prost"] }
+tonic = { workspace = true, features = ["transport", "prost", "zstd"] }
```

Comment:
Why could we enable `CompessionEncoding::Gzip` w/o having specified the feature? I guess it gets pulled in transitively? Maybe let's make it explicit here.

Reply:
Correct, let me fix that.